libs/service-portal/signature-collection/src/screens/Parliamentary/SignedList/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { ActionCard, Box, Button, Text, toast } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../../lib/messages'
+import { Modal } from '@island.is/service-portal/core'
+import { useState } from 'react'
+import { useGetSignedList } from '../../../hooks'
+import format from 'date-fns/format'
+import { useMutation } from '@apollo/client'
+import { unSignList } from '../../../hooks/graphql/mutations'
+import {
+  SignatureCollectionSignedList,
+  SignatureCollectionSuccess,
+} from '@island.is/api/schema'
+
+const SignedList = () => {
+  const { formatMessage } = useLocale()
+  const [modalIsOpen, setModalIsOpen] = useState(false)
+
+  // SignedList is typically singular, although it may consist of multiple entries, which in that case will all be invalid
+  const { signedLists, loadingSignedLists, refetchSignedLists } =
+    useGetSignedList()
+
+  const [unSign, { loading }] = useMutation(unSignList, {
+    variables: {
+      input: {
+        listId:
+          signedLists && signedLists?.length === 1
+            ? signedLists[0].id
+            : undefined,
+      },
+    },
+  })
+
+  const onUnSignList = async () => {
+    try {
+      await unSign().then(({ data }) => {
+        if (
+          (
+            data as any as {
```

Comment:
Can you do as unknown?
Will this work?
```suggestion
            data as unknown as {
```

Gotta keep those linter warnings down weeee :dancing_men:

Reply:
yes I can, yes this works